html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+<body>
+  <h1>Coding Form!</h1>
+  <form id="usrform">
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <div>
+      <input type="text" id="email">
```

Comment:
you can actually use `type="email"` here - comes baked in with some form validation!